homeassistant/components/remote/harmony.py

```diff
@@ -184,113 +194,193 @@ def name(self):
     @property
     def should_poll(self):
         """Return the fact that we should not be polled."""
-        return True
+        return False
 
     @property
     def device_state_attributes(self):
         """Add platform specific attributes."""
-        return {'current_activity': self._current_activity}
+        return {
+            ATTR_CURRENT_ACTIVITY: self._current_activity,
+            ATTR_FIRMWARE_VERSION: self._client.fw_version,
```

Comment:
No. Static info like firmware version and config version should not be part of state attributes. State attributes are meant for dynamic info related to the state of the entity.

Device info can hold this kind of static info. But it requires config entry.

https://developers.home-assistant.io/docs/en/device_registry_index.html

Reply:
Ah, Ok. I thought in having it in there due to our latest issues around firmware :-)

Config entry is something I wanna move this platform to, but not there yet. Will cancel out this PR. :-)